networkx/algorithms/dag.py

```diff
@@ -139,6 +141,14 @@ def topological_sort(G, nbunch=None, reverse=False):
 
     if nbunch is None:
         nbunch = G.nodes_iter()
+        def direct_descendants(w):
+            return G[w]
+    else:
+        nbunch_index = {x: i
+                        for i, x in enumerate(nbunch)}
```

Comment:
Google style says never to even use more than one for, but it doesn't matter anyway.